src/Models/Badge.js

```diff
@@ -0,0 +1,26 @@
+import { BADGE_BOUND, BADGE_TITLE } from '../constants/badge';
+import { MESSAGE } from '../constants/message';
+
+class Badge {
+  #eventPrice;
+
+  constructor(eventPrice) {
+    this.#eventPrice = eventPrice;
+  }
+
+  getBadgeTitle() {
```

Comment:
```suggestion
  static getBadgeTitle(eventPrice) {
```
이렇게 하면 생성자없이도 바로 클래스 메서드 사용가능할 것 같아요!